Team: allow lead to remove members

compnerd opened the pull request:
Enable team management for the lead.  Resolves #280!

compnerd committed:
Team: allow lead to remove members

Enable team management for the lead.  Resolves #280!

compnerd commented:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/63311/66276068-f3aa7c00-e843-11e9-8c7a-fb49408f1474.png">
<img width="538" alt="image" src="https://user-images.githubusercontent.com/63311/66276070-f73e0300-e843-11e9-80ef-1ac212fdcff8.png">
<img width="538" alt="image" src="https://user-images.githubusercontent.com/63311/66276072-0329c500-e844-11e9-9e7d-f72c2d404ff7.png">

samisuteria commented on Steps4Impact/Settings/TeamSettingsViewController.swift:
```diff
@@ -92,6 +95,28 @@ extension TeamSettingsViewController: SettingsActionCellDelegate {
+      alert.body = "\(name) will be permanently removed from your team"
+      alert.add(AlertAction.cancel())
+      alert.add(AlertAction(title: "Remove", style: .destructive, shouldDismiss: false) {
+        AKFCausesService.leaveTeam(fbid: fbid) { (_) in
```
I'd add a case if the result comes back as a failure from the api

samisuteria reviewed (changes requested):
Just handle failure case from api

compnerd committed:
Team: mark lead

Add lead marker on team view.  Resolves #279.

compnerd commented:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/63311/66276242-0c1b9600-e846-11e9-9400-d76437031021.png">

compnerd commented:
<img width="538" alt="image" src="https://user-images.githubusercontent.com/63311/66276256-2a819180-e846-11e9-8713-2c91180d1c84.png">

compnerd committed:
Team: handle failure of team removal

Injection of failure result from server results in alert of failure.
The member list is not altered.

compnerd committed:
Translations: add hindi string entry

samisuteria commented on Steps4Impact/Settings/TeamSettingsViewController.swift:
```diff
@@ -92,6 +95,41 @@ extension TeamSettingsViewController: SettingsActionCellDelegate {
+      alert.add(AlertAction(title: "Remove", style: .destructive, shouldDismiss: false) {
+        AKFCausesService.leaveTeam(fbid: fbid) { (result) in
+          if !result.isSuccess {
+            alert.dismiss(animated: true, completion: nil)
```
I think you need to present the new alert in the completion of this alert

samisuteria reviewed (approved):
just a nit

compnerd committed:
settings: sink presentation into completion
